src/config/config_def.c

```diff
@@ -1360,8 +1370,9 @@ static const WT_CONFIG_ENTRY config_entries[] = {
 	},
 	{ "WT_SESSION.begin_transaction",
 	  "ignore_prepare=false,isolation=,name=,priority=0,read_timestamp="
-	  ",round_to_oldest=false,snapshot=,sync=",
-	  confchk_WT_SESSION_begin_transaction, 8
+	  ",round_to_oldest=false,roundup_timestamps=(prepared=false,"
```

Comment:
Need to remove this "round_to_oldest=false" too?

Reply:
"round_to_oldest=false" is an existing one, this will be deprecated and will be removed, but not as part of this ticket.